lib/srv/db/server.go

```diff
@@ -318,7 +319,8 @@ func (s *Server) HandleConnection(conn net.Conn) {
 	// service.
 	err = s.handleConnection(ctx, tlsConn)
 	if err != nil {
-		log.WithError(err).Error("Failed to handle connection.")
+		log.WithError(err).Errorf("Failed to handle connection: %v.",
+			trace.DebugReport(err))
```

Comment:
Why cripple the stack trace?

Reply:
The stack trace is actually not being printed with `WithError`. Do you know if this was a trace issue that's fixed now? If so, debug report is not needed, otherwise I'll keep this since it's impossible to debug stuff without it.

Reply:
Yes, it should work with `WithError` since we bumped it in https://github.com/gravitational/teleport/pull/5187